docs/DeveloperGuide.md

```diff
@@ -498,6 +498,70 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
+### Adding a new student
+
+**Note:** You are encouraged to follow the test cases in order to see the expected results
+
+1. Adding a new student with all parameters specified
+
+   1. Simply type `add` into the command box and click `Enter`.
+
+   1. Copy the example shown together with the error message in the result display area.
```

Comment:
Perhaps this part can be made a little clearer

I have a suggestion:

ii. You will see an error message in the result display area that teaches you how to use the command.
iii. In the error message, an example is provided as well. Copy the example.
iv. Test case(same as the example you copied): `add n/ John .....`



will there be expected result for this as well?